theme/scripts/patronSlider.ts

```diff
@@ -0,0 +1,161 @@
+        name: "Community Patron",
+      },
+      "10": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7795092",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, patron-only events, and our archived talks earlier than the public. You'll also support our ongoing programming at Index.",
+        name: "Supporter Patron",
+      },
+      "15": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7849151",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, patron-only events, and our archived talks earlier than the public. You'll also support our ongoing programming at Index.",
+        name: "Supporter Patron+",
+      },
+      "20": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7836923",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, patron-only events, and our archived talks earlier than the public. You'll also help sustain our programming at Index.",
+        name: "Sustaining Patron",
+      },
+      "25": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7849152",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, patron-only events, and our archived talks earlier than the public. You'll also help sustain our programming at Index.",
+        name: "Sustaining Patron+",
+      },
+    },
+
+    SLIDER: null,
+
+    init() {
+      // Initialize noUiSlider instance.
+      PatronSlider.SLIDER = rangeSlider.create(
+        document.querySelector(PatronSlider.SELECTORS.slider),
+        {
+          // Start with the slider at this value.
+          start: [PatronSlider.START_VALUE],
+          // Add this many notches along the slider.
+          step: Object.keys(PatronSlider.PATRON_TIERS).length,
+          range: {
+            min: 5,
+            max: 25,
+          },
+          // Options for notches along the slider, called "pips"
+          pips: {
+            mode: rangeSlider.PipsMode.Steps,
+            filter: (value) => {
+              // If a slider value is not in our list of values,
+              // do not render a pip for that value.
+              if ([5, 10, 15, 20, 25].indexOf(value) > -1) {
+                return 1;
+              }
+
+              // Render a pip.
+              return -1;
+            },
+          },
+        }
+      );
+
+      const handle = document.querySelector(PatronSlider.SELECTORS.handle);
+      const patronTierLink = document.querySelector(
+        PatronSlider.SELECTORS.patronTierLink
+      );
+      const tierCopywriting = document.querySelector(
+        PatronSlider.SELECTORS.patronTierDescription
+      );
+      const patronTierName = document.querySelector(
+        PatronSlider.SELECTORS.patronTierName
+      );
+
+      // Used to set the content of our slider handle
```

Comment:
Up to you, but I might move this into a seprate method `intializeCopy` or something. 

I would also consider passing these domEls into the `initializeCopy` and `bindEventListeners`. A bit more functional, and wouldn't require querying the same elements twice